includes/Modules/PageSpeed_Insights.php

```diff
@@ -17,6 +17,7 @@
 use Google\Site_Kit\Core\Modules\Module_With_Deactivation;
 use Google\Site_Kit\Core\Modules\Module_With_Scopes;
 use Google\Site_Kit\Core\Modules\Module_With_Scopes_Trait;
+use Google\Site_Kit\Core\Modules\Modules;
```

Comment:
Is this needed here?

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/999698b4485ce25311ef90750c157566e1e7ada6